fsharp-backend/tests/FuzzTests/HttpClient.FuzzTests.fs

```diff
@@ -24,7 +24,7 @@ type Generator =
   static member Instant() : Arbitrary<NodaTime.Instant> = Generators.NodaTime.Instant
 
   static member String() : Arbitrary<string> =
-    // FSTODO: add in unicode
+    // FSTODO: add in unicode WHATISTHIS
```

Comment:
What's the same expectation here? is it the same expectation as in `Generators.fs`?